src/Entity/Customer.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Entity;
+
+use App\Model\Message;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ */
+class Customer
+{
+
+    /**
```

Comment:
Kiekvienas entity, kaip ir kiekviena lentelė duombazėje, turėtų turėti autoincrement primary key columną 'id'. Šiame kontekste būtų galima apsisukti ir be jo, tačiau dabartinė implementacija neveiktų, nes nė vienas fieldas neturi ORM\Id deklaracijos.